info/src/main/java/edu/artic/info/MuseumInformationViewModel.kt

```diff
@@ -46,7 +46,7 @@ class MuseumInformationViewModel @Inject constructor(
 
         dataObjectDao.getDataObject()
                 .toObservable()
-                .filterFlatMap({ it.ticketsUrl != null }, { it.ticketsUrl!! })
+                .map { it.ticketsUrlAndroid }
```

Comment:
Don't we have to filter out empty strings here?

Reply:
yes we do done.